tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/js/dialog/delete-confirm-dialog.js

```diff
@@ -38,21 +38,27 @@ define(['require', 'lodash', 'jquery', 'log', 'backbone', 'file_browser',
                     }
                 },
 
-                render: function () {
+                render: function (fileName) {
                     var self = this;
                     var fileBrowser;
                     var app = this.app;
+                    var fileTab;
                     var notification_container = this.notification_container;
-                    var workspaceServiceURL = app.config.services.workspace.endpoint;
-                    var activeTab = app.tabController.activeTab;
-                    var siddhiFileEditor = activeTab.getSiddhiFileEditor();
-                    var content = siddhiFileEditor.getContent();
-                    var providedFileName = activeTab.getTitle();
-                    var trimmedSiddhiAppName = providedFileName;
-                    if (checkEndsWithSiddhi(trimmedSiddhiAppName)) {
-                        trimmedSiddhiAppName = trimmedSiddhiAppName.slice(0, -7);
+                    // var workspaceServiceURL = app.config.services.workspace.endpoint;
+                    var providedFileName;
+                    var trimmedSiddhiAppName;
+                    //check fileName is available or not.
+                    if (!_.isNil(fileName)) {
+                        providedFileName = fileName;
+                        trimmedSiddhiAppName = trimSiddhiAppName(providedFileName);
+                        fileTab = app.tabController.getTabFromTitle(trimmedSiddhiAppName);
+                    } else {
+                        fileTab = app.tabController.activeTab;
+                        providedFileName = fileTab.getTitle();
+                        trimmedSiddhiAppName = trimSiddhiAppName(providedFileName);
                     }
-
+                    // var siddhiFileEditor = fileTab.getSiddhiFileEditor();
+                    // var content = siddhiFileEditor.getContent();
```

Comment:
Please remove this commented lines.

Reply:
80a9a2d this commit fix the above request change